src/dosbox.cpp

```diff
@@ -958,16 +963,25 @@ void DOSBOX_Init() {
 	Pmulti_remain->GetSection()->Add_string("parameters", when_idle, "");
 	Pmulti_remain->Set_help(
 	        "set type of device connected to com port.\n"
-	        "Can be disabled, dummy, modem, nullmodem, directserial.\n"
+	        "Can be disabled, dummy, serialmouse, modem, nullmodem, directserial.\n"
 	        "Additional parameters must be in the same line in the form of\n"
```

Comment:
"on the same line"

Reply:
@[johnnovak](https://github.com/johnnovak) I don't understand. What do you mean here?

Reply:
Oh, I see. The current comment says:
 - "Additional parameters must be **in** the same line ..., but should say:
 - "Additional parameters must be **on** the same line ... 

Nice catch, @johnnovak :eyes:

Reply:
Ok, changed.